docs/source/manual.rst

```diff
@@ -0,0 +1,33 @@
+User Manual for ``autojudge``
+=============================
+
+Creating your account / Logging in
+----------------------------------
+
+On the home page, click ``LOG IN`` (see top right corner).
+Login via your IITH Google account.
+
+Basic Ideas for User
+--------------------
+
+The judge uses a system of ``Contest`` s.
+A ``Contest`` consists of a set of ``Problem`` s.
+
+A user can be a ``Poster``, a ``Participant`` or neither for a ``Contest``.
+For each ``Contest`` a user will hold one and only one position.
+
+The user who creats a ``Contest`` becomes the ``Poster`` for the ``Contest``.
+The ``Poster`` can add more ``Poster`` s, who would help in creating the problems.
+These ``Poster`` s together write a set of ``Problem`` s for the ``Contest``.
+
+The ``Poster`` can leave the ``Contest`` open for all users to particpate, in which case it is called a ``Public Contest``.
+Or the ``Poster`` may specify a set of ``Participant`` s. In this case only these users can particpate in the ``Contest``. Such ``Contest`` s are also called ``Private Contest`` s.
```

Comment:
Starting a sentence with Or seems weird. Maybe add it to the previous line as continuation.